.github/workflows/docker_builds_template.yaml

```diff
@@ -0,0 +1,63 @@
+name: Docker Builds Template
```

Comment:
note: make this as a template since it will be used "release" workflow in the future